.NET/Microsoft.Recognizers.Definitions.Common/Japanese/NumbersDefinitions.cs

```diff
@@ -119,24 +120,24 @@ public static class NumbersDefinitions
       public const string PercentageRegex = @".+(?=パ\s*ー\s*セ\s*ン\s*ト)|.*(?=[％%])";
       public static readonly string DoubleAndRoundRegex = $@"{ZeroToNineFullHalfRegex}+(\.{ZeroToNineFullHalfRegex}+)?\s*[万億]{{1,2}}(\s*(以上))?";
       public const string FracSplitRegex = @"[はと]|分\s*の";
-      public const string ZeroToNineIntegerRegex = @"[一二三四五六七八九]";
+      public const string ZeroToNineIntegerRegex = @"(?:〇|一|二|三|四|五|六|七|八|九)";
       public const string NegativeNumberTermsRegex = @"(マ\s*イ\s*ナ\s*ス)";
-      public const string NegativeNumberTermsRegexNum = @"(?<!(\d+\s*)|[-－])[-－]";
+      public const string NegativeNumberTermsRegexNum = @"(?<negTerm>(-|−|－)\s*)";

```

Comment:
Be careful removing negative look-behinds as it tends to introduce false positives. Like extracting negative numbers from a phone number (ex: 555-55-55-> -55 and -55) .

Reply:
We'll check that.